pkg/sql/parser/sql.y

```diff
@@ -4508,6 +4512,22 @@ create_as_col_qualification:
   {
     $$.val = tree.NamedColumnQualification{Qualification: $1.colQualElem()}
   }
+| FAMILY family_name
+  {
+    $$.val = tree.NamedColumnQualification{Qualification: &tree.ColumnFamilyConstraint{Family: tree.Name($2)}}
+  }
+| CREATE FAMILY family_name
+  {
+    $$.val = tree.NamedColumnQualification{Qualification: &tree.ColumnFamilyConstraint{Family: tree.Name($3), Create: true}}
+  }
+| CREATE FAMILY
+  {
+    $$.val = tree.NamedColumnQualification{Qualification: &tree.ColumnFamilyConstraint{Create: true}}
+  }
+| CREATE IF NOT EXISTS FAMILY family_name
```

Comment:
I’m kinda surprised we need all these extra syntaxes — we’re creating a new table so create and in particular ifnotexists seem implied? I donno I don’t feel strongly about it

Reply:
@rolandcrosby do you have any inputs on this? This is w.r.t what column qualifiers we support when grouping into a FAMILY. I agree with David, I just copy-pasted from the qualifiers CREATE and ALTER support.

Reply:
yeah I would just support the "FAMILY" syntax in CTAS; don't think there's any reason to complicate things by adding redundant "create" or "if not exists" qualifiers for column families here.

Reply:
Done.